### PR TITLE
Manipulator controller improvements

### PR DESCRIPTION
**Pull request by pawelbudziszewski**
Some improvements to provide smoother and faster manipulator operations:

- Improved retrieving procedure: now the controller checks if the nose was retrieved before further actions (it was based on timeout previously)
- Fixed the issue that caused the nose was not retrieved when the timeout was reached and no apple was picked. Now a nose retrieval procedure is engaged in such cases.
- Fixes and tuning of manipulator physics.
- Removed some redundant settings in the UI (KrakenEffectorComponent obtains some entity ids from ManipulatorRequestBus, so the user does not need to provide them twice)
- Some minor improvements.

Signed-off-by: Paweł Budziszewski <pawel.budziszewski@robotec.ai>

### CODE REVIEW

File: Project/Gem/Source/ApplePicker/PickingStructs.h [hunk 1]
```diff
@@ -22,8 +22,9 @@ namespace AppleKraken
         PICKING_STABILIZE = 25, //!< The effector waits vacuum being built up
         RETRIEVING_NOSE = 30, //!< The effector is retrieving a fruit to storage position.
         RETRIEVING = 35, //!< The effector is retrieving a fruit to storage position.
-        RETRIEVING_STABILIZE = 40 //!< The effector is retrieving, wait for apple to drop
-
```
Comment (michalpelka) [inline Project/Gem/Source/ApplePicker/PickingStructs.h:26]: Please add state to `kMapToString`. Currently, the code crashes.

File: Project/Gem/Source/Manipulator/ManipulatorRequestBus.cpp [hunk 2]
```diff
@@ -37,6 +37,24 @@ namespace AppleKraken
...
+    }
+    AZ::EntityId ManipulatorRequestHandler::GetRestEntity()
+    {
+        AZ::EntityId p;
+        CallResult(p, FN_GetRestEntity);
+        return p;
+    }
 
 
     void ManipulatorRequestHandler::Reflect(AZ::ReflectContext* context)
```
Comment (michalpelka) [inline Project/Gem/Source/Manipulator/ManipulatorRequestBus.cpp:60]: I think that it is a good moment to drop this class with all LUA.

**Review by michalpelka — APPROVED** [review]
Tested, looks good to me.

**Comment by michalpelka** [comment]
Fixed DCO, rebased.

**Review by michalpelka — APPROVED** [review]

**Review by adamdbrw — APPROVED** [review]

**Pull request merged**